ddpui/utils/sendgrid.py

```diff
@@ -1,6 +1,8 @@
 import os
```

Comment:
It looks like we are creating an email flow implementation that is specific to Sendgrid (using specific Sendgrid API calls and using Sendgrid-specific templates). I think a better design would be to only rely on Django's native email functionality. Then, at the deployment level the application can be configured to use whatever email service is desired. You can see [here](https://docs.sendgrid.com/for-developers/sending-email/django) how to configure the Django native email functionality to work with Sendgrid.

Reply:
I agree. This was Dalgo's design and implementation so we could flag it to them and then put it in our backlog as a tech debt task we can work on.